userspace/libsinsp/filterchecks.cpp

```diff
@@ -143,6 +147,9 @@ const filtercheck_field_info sinsp_filter_check_fd_fields[] =
 	{PT_IPV4NET, EPF_NONE, PF_NA, "fd.snet", "server IP network."},
 	{PT_IPV4NET, EPF_NONE, PF_NA, "fd.lnet", "local IP network."},
 	{PT_IPV4NET, EPF_NONE, PF_NA, "fd.rnet", "remote IP network."},
+	{PT_BOOL, EPF_NONE, PF_NA, "fd.is_connected", "'true' if the socket of this event is a connected socket."},
+	{PT_CHARBUF, EPF_TABLE_ONLY | EPF_REQUIRES_ARGUMENT, PF_NA, "fd.lattr", "matches the specified kubernetes/container/host/agent/cloudProvider attribute of the local peer. E.g. 'fd.lattr.kubernetes.pod.name'."},
```

Comment:
We are using `EPF_TABLE_ONLY` to avoid even listing those filterchecks when using sysdig os. If a user peek the code and try to, they always return false.

Reply:
Oh excellent!